services/hexathons/src/routes/team.ts

```diff
@@ -154,7 +154,7 @@ teamRoutes.route("/user/:userId").get(
       return;
     }
 
-    const profiles = await ProfileModel.find({
+    const profiles = await HexathonUserModel.find({
       userId: {
         $in: team.members,
       },
```

Comment:
change this term to `hexathonUsers` and the find call should match the hexathon as well, rn hexathon users from all hexathons are taken as valid profiles